### DISCUSSION TITLE
require_once error when trying to update to 5.6.0

### DISCUSSION THREAD

**Discussion by nloshka** [firefly-iii/firefly-iii#5132] [Firefly III]
I am trying to update to the latest version on php8 but nothing works. I need help. What is missing?
Stack trace:
#0 /var/www/buh.xxx.cf/firefly-iii/public/index.php(58): require_once()
#1 {main}
  thrown in /var/www/buh.xxx.cf/firefly-iii/bootstrap/app.php on line 36" while reading response header from upstream, client: x.x.x.x, server: buh.xxx.cf, request: "GET / HTTP/2.0", upstream: "fastcgi://unix:/var/run/php-fpm/php8-fpm.sock:", host: "buh.xxx.cf"

**Comment by JC5** [comment]
Let's start with a better title to this thread.

**Reply by nloshka** [reply]
https://buh.x512.cf/p.php
Maybe I put something wrong with php8? Tell me what you need to subtract or add to php8?
Thanks

**Reply by JC5** [reply]
8.0.2 should be the correct version.
